docs/data/material/migration/migrating-to-v6/migrating-to-v6.md

```diff
@@ -258,54 +275,62 @@ npx @mui/codemod@next v6.0.0/grid-v2-props <path/to/folder>
 You need to modify the import from `@mui/material/Unstable_Grid2` to `@mui/material/Grid2` before running the codemod.
 :::
 
-If you have custom breakpoints, the change is the same:
+##### Using custom breakpoints
+
+The usage described above also applies to custom breakpoints:
 
 ```diff
 -<Grid mobile={12} mobileOffset={2} desktop={6} desktopOffset={4}>
 +<Grid size={{ mobile: 12, desktop: 6 }} offset={{ mobile: 2, desktop: 4 }}>
 ```
 
-Which you can cover with the same codemod by providing the custom breakpoints as an argument:
+You can use the same codemod for custom breakpoints by providing the breakpoints as an argument:
 
 ```bash
 npx @mui/codemod@next v6.0.0/grid-v2-props <path/to/folder> --jscodeshift='--muiBreakpoints=mobile,desktop'
 ```
 
-#### Removal of the disableEqualOverflow prop
+#### disableEqualOverflow prop removed
 
-Previously, the Grid overflowed its parent.
-In v6, this is fixed, with the Grid being contained inside its parent's padding:
+In v5, the Grid overflowed its parent.
+In v6 the Grid is correctly contained within its parent's padding:
 
 <img src="/static/material-ui/migration-v5/grid-overflow-change.png" style="width: 814px;" alt="Before and after of the Grid no longer overflowing its parent in v6." width="1628" height="400" />
 
-This removes the need for the `disableEqualOverflow` prop:
+This eliminates the need for the `disableEqualOverflow` prop:
 
 ```diff
 -<Grid disableEqualOverflow>
 +<Grid>
 ```
 
-#### Spacing is no longer considered inside the Grid item's box
+:::warning
+This update may lead to unexpected changes to your app's layout.
+Still, we strongly recommend adopting this new behavior rather than trying to replicate the old pattern, as the new version is more predictable and modern.
+:::
+
+#### Grid item spacing change
 
-Previously, Grid items included spacing in their boxes.
-In v6, this is fixed:
+In v5, Grid items included spacing in their boxes.
+In v6, Grid items no longer inclue spacing in their boxes.
 
-<img src="/static/material-ui/migration-v5/grid-spacing-change.png" style="width: 814px;" alt="Before and after of the Grid items no longer including spacing in their box." width="1628" height="400" />
+Note that the item position doesn't change.
 
 :::warning
```

Comment:
I would have only one of these warnings, as it's the same warning for this and the previous section. If you feel strongly about having it in both sections, then let's move this one bellow the image so it's structured in the same way as the previous section.